### DISCUSSION TITLE
Issues about line_params in MLineVertex

### DISCUSSION THREAD

**Discussion by maslke** [mozman/ezdxf#1135] [Q&A]
Hello everyone, I have a question about the MLine structure and line_params params.

I have a DXF file that has Mline structure. When reading with the ezdxf library, for the Mline structure, ezdxf provides the following parameters:
<img width="1355" alt="image" src="https://github.com/user-attachments/assets/efa54fb8-c0dc-473f-8521-4bc5e168651f">

It can be seen that in line_params, the dash-length data for the two part of styles is the same. (I think it should not be the same, because the start point of line has an offset.)

With these parameters, I have converted the DXF file into a geojson file. When previewed with QGIS, it can be observed that there is a coordinate offset in the Mline data:
<img width="538" alt="image" src="https://github.com/user-attachments/assets/5ca838a3-f471-4585-94b3-3b31c47b0e27">

I have tested in version of "1.1.0" and "1.3.0" and the issue is same.

Is this an issue with ezdxf, or is there a misunderstanding on my part regarding line_params?

Additionally, when previewing the aforementioned DXF file with the `ezdxf view` method, the preview is as follows:
![image](https://github.com/user-attachments/assets/2ea18911-ed1d-4206-b1f5-19aadb35ae55)

![image](https://github.com/user-attachments/assets/c30643d8-3fdd-4708-ae5c-9db42902157c)

However, the same DXF file, when previewed in QGIS, looks like this:
![image](https://github.com/user-attachments/assets/10f9208f-e3f6-48e4-8e26-957098956b83)

In my case, QGIS provides a more accurate representation. The discrepancy arises during the parsing of the MLine entity. Is this an issue with the "ezdxf view" functionality?

I greatly appreciate any help.

**Comment by mozman** [comment]
If you provide your data, I might take a look at it.

**Reply by maslke** [reply]
[dc.zip](https://github.com/user-attachments/files/16407835/dc.zip)

Thanks a lot, I really appreciate it.

**Reply by maslke** [reply]
Finally, I converted the DXF file to a JSON file using ezdxf and achieved the same visual style as in AutoCAD.

I misunderstood the line_params earlier.

Thank you.
